### PR TITLE
[full-ci] [tests-only] Added tests to list sharedWithMe for file with same name shared by multiple users

### PR DESCRIPTION
**Pull request by grgprarup**
## Description
Added scenario:
-  Scenario: user lists the file with same name shared by two users with him/her
-   Scenario: user lists the folder with same name shared by two users with him/her

## Related Issue

- Part of https://github.com/owncloud/ocis/issues/7826

## How Has This Been Tested?

- Locally
- :robot:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by amrita-shrestha — APPROVED** [review]
lgtm

**Review by SagarGi — COMMENTED** [review]
LGTM :+1:

File: tests/acceptance/features/apiSharingNg/sharedWithMe.feature [hunk 1]
```diff
@@ -1951,3 +1951,591 @@ Feature: an user gets the resources shared to them
...
+                      "type": "boolean",
+                      "enum": [true]
+                    },
+                    "createdBy": {
+                      "type": "object",
+                      "required": ["user"],
+                      "properties": {
+                        "user": {
+                          "type": "object",
+                          "required": ["displayName", "id"],
```
Comment (nabim777) [inline tests/acceptance/features/apiSharingNg/sharedWithMe.feature:2013]:
```suggestion
                          "required": ["displayName", "id"],
```
if id not needed to check then can be removed from required

File: tests/acceptance/features/apiSharingNg/sharedWithMe.feature [hunk 2]
```diff
@@ -1951,3 +1951,591 @@ Feature: an user gets the resources shared to them
...
+              "oneOf": [
+                {
+                  "type": "object",
+                  "required": [
+                    "@UI.Hidden",
+                    "@client.synchronize",
+                    "createdBy",
+                    "eTag",
+                    "file",
+                    "id",
```
Comment (nabim777) [inline tests/acceptance/features/apiSharingNg/sharedWithMe.feature:1992]:
```suggestion
```
if not needed to check then we can remove them

**Review by nabim777 — APPROVED** [review]

File: tests/acceptance/features/apiSharingNg/sharedWithMe.feature [hunk 3]
```diff
@@ -1951,3 +1951,591 @@ Feature: an user gets the resources shared to them
...
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": ["value"],
+        "properties": {
+          "value": {
+            "type": "array",
+            "items": {
+              "oneOf": [
```
Comment (saw-jan) [inline tests/acceptance/features/apiSharingNg/sharedWithMe.feature:1983]:
```suggestion
            "minItems": 2,
            "maxItems": 2,
            "items": {
              "oneOf": [
```

File: tests/acceptance/features/apiSharingNg/sharedWithMe.feature [hunk 4]
```diff
@@ -1951,3 +1951,591 @@ Feature: an user gets the resources shared to them
...
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": ["value"],
+        "properties": {
+          "value": {
+            "type": "array",
+            "items": {
```
Comment (saw-jan) [inline tests/acceptance/features/apiSharingNg/sharedWithMe.feature:2276]:
```suggestion
            "type": "array",
            "minItems": 2,
            "maxItems": 2,
            "items": {
```

**Review by saw-jan — CHANGES_REQUESTED** [review]
let's use `minItems` and `maxItems` to make more stricter checks

**Review by saw-jan — APPROVED** [review]
looking good :sparkles:

**Pull request merged**